libr/include/r_io.h

```diff
@@ -158,6 +158,7 @@ typedef struct r_io_plugin_t {
 	char *author;
 	char *license;
 	void *widget;
+	char *prefixes;
```

Comment:
thats wrong, this buffer cant be modified, use const char *.. or const char ** to use {"foo","bar",NULL}

Reply:
I will change it to a const buffer.